openassessment/xblock/test/test_openassessment.py

```diff
@@ -91,95 +95,211 @@ def test_student_view_workflow_error(self, xblock):
             xblock_fragment = self.runtime.render(xblock, "student_view")
 
         # Expect that the page renders even if the update fails
-        self.assertTrue(xblock_fragment.body_html().find("Openassessmentblock"))
+        self.assertIn("OpenAssessmentBlock", xblock_fragment.body_html())
 
-    @scenario('data/dates_scenario.xml')
```

Comment:
Can you try deleting `data/dates_scenario.xml` and see if tests continue to pass? If we're no longer using this file, it should be removed.